windows/windows_update_install/prepare_msu_file.yml

```diff
@@ -1,30 +1,41 @@
 # Copyright 2023-2024 VMware, Inc.
 # SPDX-License-Identifier: BSD-2-Clause
 ---
+# Copy the .msu file of type msu_file_type from the shared nfs to guest OS.
+# Parameters:
+#   msu_file_type: SSU or LCU.
+# Return:
+#  msu_file_path: the .msu file path in guest OS.
+#
+- name: "Initialize the .msu file path and file name regexp"
+  ansible.builtin.set_fact:
+    msu_root_dir: "C:\\msu"
+    msu_file_name_reg: "{{ '*x64*' if guest_os_ansible_architecture == '64-bit' else '*x86*'}}"
+    msu_file_path: ""
+
```

Comment:
Better to check `msu_file_type` is set or set it with a default value when it is undefined.

Reply:
Will add assert to check if it's set. thanks.